build.gradle

```diff
@@ -48,7 +48,7 @@ publishing {
 repositories {
     mavenLocal()
     // Remove the commented code below once TransportService is published to maven
-    //maven { url "https://aws.oss.sonatype.org/content/repositories/snapshots" }
+    maven { url "https://aws.oss.sonatype.org/content/repositories/snapshots" }
```

Comment:
You should be able to back this out now that you are working off the feature/extensions published locally.